rpm/sailfish-browser.spec

```diff
@@ -31,6 +31,7 @@ Requires: sailfish-browser-settings = %{version}
 Requires: qt5-plugin-imageformat-ico
 Requires: qt5-plugin-imageformat-gif
 Requires: qt5-plugin-position-geoclue
+Requires: mapplauncherd >= 4.1.17
```

Comment:
If this is newer than what is available currently for users, target this PR to the next branch.